SI/Sem.04/Pract.04/Task03.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+
+using namespace std;
+
+bool IsPositive(int num)
+{
+	if (num > 0)
+	{
+		return true;
+	}
+	else
+	{
+		return false;
+	}
+}
+
+bool IsNegative(int num)
+{
+	if (num < 0)
```

Comment:
num < 0 is a boolean expression, so it returns a boolean value that you can directly return